sklearn/ensemble/weight_boosting.py

```diff
@@ -863,9 +864,10 @@ class AdaBoostRegressor(BaseWeightBoosting, RegressorMixin):
 
     Parameters
     ----------
-    base_estimator : object, optional (default=DecisionTreeRegressor)
+    base_estimator : object, optional (default=None)
         The base estimator from which the boosted ensemble is built.
-        Support for sample weighting is required.
+        Support for sample weighting is required. If we chose the default value,
```

Comment:
The number of characters in a single line is not allowed to be over 79. Please add a newline between "default" and "value"